### ISSUE TITLE
does this integreation use a github account?

### ISSUE THREAD

**Issue by mccannical** [atlassian/github-for-jira#803]
I do not believe this is the case, but, has this always been the case?

we likely have a left over account on my end from some internal testing, but, id like to validate its not in use by this integration

**Comment by mboudreau** [comment]
@mccannical so sorry, this question slipped through the cracks.

**TLDR; We only use your personal account to install the app on an organization.  After that everything is done through the Github App API.**

When you install a github app, you need to use your account to do so.  You can only install the app on organizations _you have admin access to_.  You need to accept the permissions that the app is requesting.

Once that is done, there's a transaction that happens between github and our app where we keep an installation id which is unique to each organization we are installed on.  The app itself is already registered on github and secured through a unique private key only available to our app.  From this point forward, everything the app does is based on this installation id and the private key to be able to call Github APIs on your organization's behalf.  Think of it like the app being it's own separate user which you've allowed X access to your organizational repos and we use that identity for all interactions.

I hope that answers your question.

**Comment by rachellerathbone** [comment]
Hey @mccannical. As it's been several weeks with no activity on this issue, I'm going to go ahead and close it. Please feel free to reopen and ask more questions if you need further clarification.
